Tools/bash/install.sh

```diff
@@ -0,0 +1,562 @@
+#!/bin/bash
+#
+# Bebop: an extremely simple, fast, efficient, cross-platform serialization format
+# this script is responsible for installing /updating the Bebop compiler
+# https://github.com/rainwayapp/bebop
+#
+# The Apache License 2.0 License
+#
+# Copyright (c) 2017 Rainway, Inc.
+
+set -u
+
+abort() {
+    printf "%s\n" "$@"
+    exit 1
+}
+
+# Fail fast with a concise message when not using bash
+# Single brackets are needed here for POSIX compatibility
+if [ -z "${BASH_VERSION:-}" ]; then
+    abort "Bash is required to use this script."
+fi
+
+### Constants
+readonly BEBOPC_VERSION="${1:-2.3.1}"
```

Comment:
Do we have to hardcode this here? It will make it much more likely that we will forget to update it. Maybe we could source https://raw.githubusercontent.com/RainwayApp/bebop/master/.env

Reply:
this should be updated in the release pipeline before pushing it to S3